tests/mappers/test_archive_extractor.py

```diff
@@ -18,38 +18,46 @@
 from xml.etree import ElementTree as ET
 
 from o2a.utils.file_archive_extractors import ArchiveExtractor
+from o2a.o2a_libs.property_utils import PropertySet
 
 
 class TestArchiveExtractor(unittest.TestCase):
     def setUp(self):
-        self.default_params = {
+        self.configuration_properties = {}
+        self.job_properties = {
             "nameNode": "hdfs://",
             "oozie.wf.application.path": "hdfs:///user/pig/examples/pig_test_node",
         }
+        self.action_node_properties = {}
+        self.property_set = PropertySet(
+            configuration_properties=self.configuration_properties,
+            job_properties=self.job_properties,
+            action_node_properties=self.action_node_properties,
+        )
 
     def test_add_relative_archive(self):
         # Given
-        archive_extractor = ArchiveExtractor(oozie_node=Element("fake"), params=self.default_params)
+        archive_extractor = ArchiveExtractor(oozie_node=Element("fake"), property_set=self.property_set)
         # When
         archive_extractor.add_archive("test_archive.zip")
         # Then
-        self.assertEqual(archive_extractor.archives, ["test_archive.zip"])
+        self.assertEqual(["test_archive.zip"], archive_extractor.archives)
```

Comment:
What do you think about taking this change into a separate PR/commit?  The current commit is very large.

Reply:
It will not change the size of the commit heavily. What exactly do you propose to split :)?

Reply:
Make it easy to make reviews, but it's not required.